### PR TITLE
Rewrite of the abstract.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Abstract doesn't really captured what was inside the document. It also contains a bunch of spelling mistakes, etc.

### CODE REVIEW

**Comment by mounirlamouri** [comment]
Could you try to keep a 80 characters limitation. It is a pain to read with the review tool (it doesn't do line break).

**Comment by marcoscaceres** [comment]
Sure, will fix. 

Something to add to the process doc.

File: index.html [hunk 1]
```diff
@@ -45,16 +45,45 @@
...
+    various security related permission requests to resources on the Web and/or
+    services and capabilities on the device.</p>
+
+    <figure>
+      <img src="images/apptypes.png" alt="">
+
+      <figcaption>
+        The figure shows the overlap between traditional web applications,
+        hosted web applications, and packaged applications. Web applications
+        make use of standard Web technologies but don't make use of of an
```
Comment (ming-jin) [inline index.html]: Typo: "of of"

**Comment by marcoscaceres** [comment]
Fixed.

File: index.html [hunk 2]
```diff
@@ -45,16 +45,45 @@
...
+        of an application manifest, while packaged applications have both an
+        application manifest and are packaged using [[!ZIP]].
+      </figcaption>
+    </figure>
+
+    <p>Installation of hosted or packaged applications onto a device can be
+    initiated through either linking to an application manifest or through the
+    use of the <code>ApplicationRegistry</code> API. This API gives a developer
+    a degree of control over the installation process, as well as providing a
+    means to programmatically unistall an application.</p>
```
Comment (ming-jin) [inline index.html]: In fact "uninstall" is interface of Application interface, but not ApplicationRegistry interface.

Comment (marcoscaceres) [inline index.html]: ah, well spotted. Will remove "as well as providing a means to programmatically unistall an application."

**Pull request merged**